orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/TaskController.groovy

```diff
@@ -465,26 +460,19 @@ class TaskController {
   PipelineExecution updatePipelineStage(
     @PathVariable String id,
     @PathVariable String stageId, @RequestBody Map context) {
-    def pipeline = executionRepository.retrieve(PIPELINE, id)
-    def stage = pipeline.stages.find { it.id == stageId }
-    if (stage) {
-      stage.context.putAll(context)
-      validateStageUpdate(stage)
-
-      stage.lastModified = new StageExecution.LastModifiedDetails(
-        user: AuthenticatedRequest.getSpinnakerUser().orElse("anonymous"),
-        allowedAccounts: AuthenticatedRequest.getSpinnakerAccounts().orElse(null)?.split(",") ?: [],
-        lastModifiedTime: System.currentTimeMillis()
-      )
-
-      // `lastModifiedBy` is deprecated (pending a update to deck)
-      stage.context["lastModifiedBy"] = AuthenticatedRequest.getSpinnakerUser().orElse("anonymous")
-
-      executionRepository.storeStage(stage)
-
-      executionRunner.reschedule(pipeline)
-    }
-    pipeline
+    executionOperator.updateStage(PIPELINE, id, stageId,
```

Comment:
can you add a return statement here please, i just spent 5 minutes utterly confused where is the return from `doInternal` used... thought i knew it should be here it took me a while to "remember"... ahhh... groovy

```suggestion
    return executionOperator.updateStage(PIPELINE, id, stageId,
```